tests/resources/requests/time_filter.json

```diff
@@ -0,0 +1,75 @@
+{
+  "locations": [
+    {
+      "id": "London center",
+      "coords": {
+        "lat": 51.50893,
+        "lng": -0.131387
+      }
+    },
+    {
+      "id": "Hyde Park",
+      "coords": {
+        "lat": 51.508824,
+        "lng": -0.167093
+      }
+    },
+    {
+      "id": "ZSL London Zoo",
+      "coords": {
+        "lat": 51.536067,
+        "lng": -0.153596
+      }
+    }
+  ],
+  "departure_searches": [
+    {
+      "id": "forward search example",
+      "arrival_location_ids": [
+        "Hyde Park",
+        "ZSL London Zoo"
+      ],
+      "departure_location_id": "London center",
+      "departure_time": "2022-11-24T12:00:00",
```

Comment:
This will fail when map expires. When you have a date you should put `NOW`

Reply:
I mocked these requests, so it won't fail

Reply:
This time I decided to separate

Reply:
We still need integration tests, there is a chance that SDK won't work with API and we have to catch it

Reply:
Yeah, I will add them with a new pr